v-next/hardhat-ignition-viem/src/index.ts

```diff
@@ -1,7 +1,25 @@
-import "@ignored/hardhat-vnext-ignition";
-import "@ignored/hardhat-vnext-viem";
+import type { HardhatPlugin } from "@ignored/hardhat-vnext/types/plugins";
 
-import "./type-extensions";
+import hardhatIgnitionPlugin from "@ignored/hardhat-vnext-ignition";
+import hardhatViemPlugin from "@ignored/hardhat-vnext-viem";
+
+import { PLUGIN_ID } from "./internal/constants.js";
+
+import "./type-extensions.js";
+
+const hardhatIgnitionViemPlugin: HardhatPlugin = {
+  id: PLUGIN_ID,
+  npmPackage: "@ignored/hardhat-vnext-ignition-viem",
+  dependencies: [
+    async () => hardhatIgnitionPlugin,
```

Comment:
Switch to dynamic import - this can be done later.

Reply:
I have added a post initial public alpha issue for this here: https://github.com/NomicFoundation/hardhat/issues/6115?issue=NomicFoundation%7Chardhat%7C6289